### PR TITLE
WMCO 8.0.0 Release Notes

### PR DESCRIPTION
**Pull request by mburke5678**
The bulk of the [changes are here](https://github.com/openshift/openshift-docs/pull/57120/files#diff-600636c782529c89d3a17a3c4212d8f7f64f243186adc9975179c069d72858acR32-R54). The rest of the [windows-containers-release-notes-8-x.adoc](https://github.com/openshift/openshift-docs/pull/57120/files#diff-600636c782529c89d3a17a3c4212d8f7f64f243186adc9975179c069d72858ac) file is template.

**NOTE:** The link to the errata will 404 until the errata is officially published (at GA).

Preview: [Release notes for Red Hat Windows Machine Config Operator 8.0.0](https://57120--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes-8-x.html#wmco-8-0-0)

### CODE REVIEW

File: windows_containers/windows-containers-release-notes-8-x.adoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+For more information, see the Red Hat OpenShift Container Platform Life Cycle Policy document for link:https://access.redhat.com/support/policy/updates/openshift#windows[{productwinc}].
+
+If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
+endif::openshift-origin[]
+
+[id="wmco-8-0-0"]
+== Release notes for Red Hat Windows Machine Config Operator 8.0.0
+
+This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 8.0.0 were released in link:https://access.redhat.com/errata/RHSA-2023:1372[RHSA-2023:1372].
```
Comment (jfrancoa) [inline windows_containers/windows-containers-release-notes-8-x.adoc:34]: I tried accessing the [errata link](https://access.redhat.com/errata/RHSA-2023) but I got a 404 , is that ok at this point in time?

Comment (jfrancoa) [reply windows_containers/windows-containers-release-notes-8-x.adoc:34]: no problem, I just wanted to confirm if that was the case. Thanks for the quick answer.

**Comment by mburke5678** [comment]
**@peer-review-squad**  The bulk of the [changes are here](https://github.com/openshift/openshift-docs/pull/57120/files#diff-600636c782529c89d3a17a3c4212d8f7f64f243186adc9975179c069d72858acR32-R54). The rest of the [windows-containers-release-notes-8-x.adoc](https://github.com/openshift/openshift-docs/pull/57120/files#diff-600636c782529c89d3a17a3c4212d8f7f64f243186adc9975179c069d72858ac) file is template.

NOTE: The link to the errata will 404 until the errata is officially published (at GA).

File: windows_containers/windows-containers-release-notes-8-x.adoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+==== Support for the pod `os` parameter
+
+You can now use the `spec.os.name.windows` parameter in your workload pods to authoritatively identify the pod operating system for validation and to enforce Windows-specific pod security context constraints (SCCs). It is recommended that you configure this parameter in your workload pods. 
+
+For more information, see xref:../windows_containers/scheduling-windows-workloads.adoc#sample-windows-workload-deployment_scheduling-windows-workloads[Sample Windows container workload deployment].
+
+[id="wmco-8-0-0-must-gather"]
+==== WICD logs are added to must-gather
+
+The must-gather tool now collects the service logs generated by the Windows Instance Config Daemon (WICD) from Windows nodes.
```
Comment (snarayan-redhat) [inline windows_containers/windows-containers-release-notes-8-x.adoc:49]: I see must-gather is put in `` in our documents. Eg. [here](https://docs.openshift.com/container-platform/4.12/virt/logging_events_monitoring/virt-collecting-virt-data.html) and [here](https://docs.openshift.com/container-platform/4.12/authentication/understanding-and-managing-pod-security-admission.html#security-context-constraints-psa-alert-eval_understanding-and-managing-pod-security-admission)
```suggestion
The `must-gather` tool now collects the service logs generated by the Windows Instance Config Daemon (WICD) from Windows nodes.
```

File: windows_containers/windows-containers-release-notes-8-x.adoc [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+[id="wmco-8-0-0-bug-fixes"]
+=== Bug fixes
+
+* Previously, the test to determine if the Windows Defender antivirus service is running incorrectly checked for any process whose name started with `Windows Defender`, regardless of state. This resulted in an error when the WMCO created firewall exclusions for containerd on instances without `Windows Defender` installed. This fix now checks for the presence of the specific running process associated with the Windows Defender antivirus service. As a result, the WMCO can properly configure Windows instances as nodes regardless of whether Windows Defender is installed. (link:https://issues.redhat.com/browse/OCPBUGS-1513[*OCPBUGS-1513*])
```
Comment (snarayan-redhat) [inline windows_containers/windows-containers-release-notes-8-x.adoc:54]:
```suggestion
* Previously, the test to determine if the Windows Defender antivirus service is running incorrectly, checked for any process whose name started with `Windows Defender`, regardless of state. This resulted in an error when the WMCO created firewall exclusions for containerd on instances without `Windows Defender` installed. This fix now checks for the presence of the specific running process associated with the Windows Defender antivirus service. As a result, the WMCO can properly configure Windows instances as nodes regardless of whether Windows Defender is installed. (link:https://issues.redhat.com/browse/OCPBUGS-1513[*OCPBUGS-1513*])
```

**Review by snarayan-redhat — COMMENTED** [review]
Great work! Just couple of nits.

**Review by jfrancoa — COMMENTED** [review]
/lgtm

File: windows_containers/windows-containers-release-notes-8-x.adoc [hunk 4]
```diff
@@ -53,6 +53,8 @@ The `must-gather` tool now collects the service logs generated by the Windows In
 
 * Previously, the test to determine if the Windows Defender antivirus service is running was incorrectly checking for any process whose name started with `Windows Defender`, regardless of state. This resulted in an error when the WMCO created firewall exclusions for containerd on instances without `Windows Defender` installed. This fix now checks for the presence of the specific running process associated with the Windows Defender antivirus service. As a result, the WMCO can properly configure Windows instances as nodes regardless of whether Windows Defender is installed. (link:https://issues.redhat.com/browse/OCPBUGS-1513[*OCPBUGS-1513*])
 
+* Previously, in-tree storage was not working for Windows nodes on VMware vSphere. With this fix, {productwinc} properly supports in-tree storage for all cloud providers.  (link:https://issues.redhat.com/browse/WINC-1014[*WINC-1014*])
+
```
Comment (mburke5678) [inline windows_containers/windows-containers-release-notes-8-x.adoc:57]: @sebsoto Can you PTAL at this commit to add your in-tree storage fix to the 8.0.0 release notes?

**Pull request merged**